perf/specjbb/run/options/specjbb-multijvm-simple.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+export GROUP_COUNT=1
```

Comment:
Could do with some comments as to why we are using these settings

Reply:
I've added comments to the shell scripts and fixed the spacing issue in the xml files.